### DISCUSSION TITLE
Bfetchporn

### DISCUSSION THREAD

**Discussion by NNBnh** [NNBnh/bfetch#1] [Show and tell]
<h1 align="center"><i>Bfetchporn</i></h1>
<p align="center">The home for SuperB Fetch customization!</p>

<p align="center"><img width="100%" src="https://user-images.githubusercontent.com/43980777/107769286-5a11a980-6d6a-11eb-9d65-ed07bf79a5c0.png"></p>

### Submit screenshots of all your Bfetch cool, nifty and *superb* configurations.<br>Show the world how sexy your fetch can be!

**Comment by NNBnh** [comment]
Repost:

[![asciicast](https://asciinema.org/a/381349.svg)](https://asciinema.org/a/381349)

[![asciicast](https://asciinema.org/a/379073.svg)](https://asciinema.org/a/379073)

**Comment by NNBnh** [comment]
Image i post on [This Reddit post](https://www.reddit.com/r/unixporn/comments/ko9my8/oc_bfetch_dynamic_fetching_tool_that_superb):

![Reddit](https://i.redd.it/79hyehzedp861.png)

**Comment by NNBnh** [comment]
Video:

https://user-images.githubusercontent.com/43980777/106760642-9ce6c980-6666-11eb-9dba-3ef377c894a8.mp4

**Comment by NNBnh** [comment]
Move SuperB Fetch images here:

| |[Neofetch](https://github.com/dylanaraps/neofetch)|[Pfetch](https://github.com/dylanaraps/pfetch)|
|:-:|-|-|
|Without SuperB Fetch|![Neofetch](https://user-images.githubusercontent.com/43980777/108312895-ef3cf400-71e9-11eb-8783-5d5d437c166a.png)|![Pfetch](https://user-images.githubusercontent.com/43980777/108312916-f532d500-71e9-11eb-9313-66363806bb4e.png)|
|With SuperB Fetch|![Bfetch](https://user-images.githubusercontent.com/43980777/108312905-f237e480-71e9-11eb-97db-4f2d0f5cdb8b.png)|![Bfetch](https://user-images.githubusercontent.com/43980777/108312925-f82dc580-71e9-11eb-97ab-1d9becab58dd.png)|

|Paper mode|Classic mode|
|-|-|
|![Paper mode](https://user-images.githubusercontent.com/43980777/108312884-eba96d00-71e9-11eb-90f7-301537ba2000.png)|![Classic mode](https://user-images.githubusercontent.com/43980777/108312871-e64c2280-71e9-11eb-8569-f38765891480.png)|

**Comment by K4zoku** [comment]
![image](https://user-images.githubusercontent.com/34259386/148899999-15a971e3-8ce2-48b9-bdb8-edffc9c3d287.png)
